docs/deciding_what_data_to_stream_with_dfe_analytics.md

```diff
@@ -0,0 +1,70 @@
+
+## What is the basis for processing personal data in BigQuery?
+```dfe-analytics``` and ```dfe-analytics-dataform``` have been designed with the ['public task' GDPR basis for processing](https://ico.org.uk/for-organisations/uk-gdpr-guidance-and-resources/lawful-basis/a-guide-to-lawful-basis/public-task/) in mind. They are not suitable for use with data that is stored and processed on [another legal basis](https://ico.org.uk/for-organisations/uk-gdpr-guidance-and-resources/lawful-basis/a-guide-to-lawful-basis/), for example consent, contract or legitimate interest. If personal data has been collected by a digital service on another legal basis it should **not** be streamed with ```dfe-analytics``` without further assurance and engineering. This work would need to ensure that if consent is withdrawn or there is a request for deletion of a user's data, it is possible to delete the full change history of this user's records in the database in BigQuery, as well as all their web traffic data.
+
+## How do we exclude data from streaming?
+Data for which DfE has no basis to store or process **must not** be streamed into BigQuery by ```dfe-analytics```.
+
+This data may take the form of individual fields within a table in an application's database, or entire tables.
+
+To exclude these fields, remove them from ```config/analytics.yml``` and add them to ```config/analytics_blocklist.yml```.
+
+For more information see [here](https://github.com/DFE-Digital/dfe-analytics/tree/SL-dev?tab=readme-ov-file#5-send-database-events).
+
+## How long should we keep data for?
+Data processed under the 'public task' basis for processing described in the previous section may continue to be stored and processed for as long as it is needed for any of the public task(s) that is stored for.
+
+This means that once it is no longer useful for any such task it **must** be deleted. For example:
+- Data to support analysis of teachers' progression throughout their careers so that the effectiveness of policy measures to retain teachers can be evaluated might need to be retained for 50 years
+- Data that is only stored to support evaluation of the impact of policy changes on ITT recruitment might need to be stored for a number of years (say, 7) but not longer
+- Data that is only stored to support evaluation of the immediate effects of a design or content change to a digital service might only need to be stored for around a year
+
+IAOs and their teams should also consider public tasks which may need to be performed by other teams within DfE, both now and in the future, when evaluating how long data should be retained for.
+
+Teams **should** consider [implementing a data retention schedule](https://github.com/DFE-Digital/dfe-analytics-dataform/?tab=readme-ov-file#data-retention-schedules) in ```dfe-analytics-dataform``` to ensure that data streamed by ```dfe-analytics``` is deleted automatically once it is no longer required.
```

Comment:
Should we flag that we've delivered functionality to enable actioning of data retention schedules? 

Can we make it clear that teams use this functionality themselves, to implement their data retention schedules. 
Who do teams need to seek advice and signoff for their data retention schedules? Is this their IAO?

Does this guidance (https://github.com/DFE-Digital/dfe-analytics-dataform/?tab=readme-ov-file#data-retention-schedules) cover all the things a team needs to do to implement their data retention schedule?

Reply:
Added "It is the IAO's ultimate responsibility to sign off on this."

I think it's already clear that teams use this functionality themselves from the sentence "Teams **must** consider [implementing a data retention schedule](https://github.com/DFE-Digital/dfe-analytics-dataform/?tab=readme-ov-file#data-retention-schedules) in ```dfe-analytics-dataform```..."

The guidance does cover all the things a team needs to do to implement their data retention schedule but could use some attention - I'll attempt this as a separate PR on that repo.